src/tool_operate.c

```diff
@@ -1658,6 +1662,31 @@ static CURLcode single_transfer(struct GlobalConfig *global,
             break;
         }
 
+#ifdef CURL_CA_EMBED
+        if(!config->cacert && !config->capath) {
+          struct curl_blob blob;
+          blob.data = (void *)curl_ca_embed;
+          blob.len = strlen((const char *)curl_ca_embed);
+          blob.flags = CURL_BLOB_NOCOPY;
+          result = curl_easy_setopt(curl, CURLOPT_CAINFO_BLOB, &blob);
```

Comment:
Maybe we should log here in verbose mode? Currently the log from this is `successfully imported CA certificate blob` from `lib/vtls/openssl.c` and `lib/vtls/wolfssl.c`, but it will be good if it's more clear we're using a bundled CA (and not from somewhere else)

Reply:
Agreed this would be a nice touch. But how to make a verbose output from `src`? I could not figure it out.

Reply:
Ah, perhaps `notef()`?

Reply:
> Ah, perhaps `notef()`?

Hmm, Yeah that seems correct

Reply:
Pushed a commit with that call.